erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -245,32 +280,74 @@ def on_cancel(self):
 	def on_trash(self):
 		delete_events(self.doctype, self.name)
 
+	
+	@frappe.whitelist()
```

Comment:
One line space is enough
```suggestion
		delete_events(self.doctype, self.name)

	@frappe.whitelist()
```